packages/teams-js/prepBetaRelease.js

```diff
@@ -167,13 +176,17 @@ function getNewPkgJsonContent(currNextDevVer) {
   return packageJson;
 }
 
-function prepNewDevRelease(devStdout) {
+function prepBetaRelease(devStdout) {
   const newPackageJson = getNewPkgJsonContent(devStdout);
   const newVersion = newPackageJson.version;
   saveJsonFile(newPackageJson);
   return newVersion;
 }
 
 (() => {
-  exec(`npm view @microsoft/teams-js version --tag next-dev`).then(({ stdout, stderr }) => prepNewDevRelease(stdout.trim()));
+  exec(`yarn beachball bump`).then(
```

Comment:
I think if you change this to the following the code works. But the comment about the first time we do this still applies.

```
`exec(`pushd ../.. && yarn beachball bump`).then(`
```

Reply:
I should amend my previous comment. When I make the change above and run the script, I can see that `yarn beachball bump` no longer throws errors. But, it doesn't seem to be updating the package.json (though it's definitely running and updating the change log) before running the next command.

Reply:
> I think if you change this to the following the code works. But the comment about the first time we do this still applies.
> 
> ```
> `exec(`pushd ../.. && yarn beachball bump`).then(`
> ```

I've tried `pushd`, but for some reason it complains that my syntax isn't correct. Changing it to `cd` fixes it however.